src/maps/filters.ts

```diff
@@ -556,7 +541,14 @@ export const mapWorkPoolWorkersFilter: MapFunction<WorkPoolWorkersFilter, WorkPo
 
 export const mapWorkPoolQueuesFilter: MapFunction<WorkPoolQueuesFilter, WorkPoolQueuesFilterRequest> = function(source) {
   return removeEmptyObjects({
-    work_pools: this.map('WorkPoolFilter', source.workPools, 'WorkPoolFilterRequest'),
+    work_queues: {
```

Comment:
😱 - this was actually wrong before. the Read Work Queues API that's under Work Pools uses a _work queue_ filter here (https://app.stg.prefect.dev/api/docs#tag/Work-Pools/operation/read_work_queues_api_accounts__account_id__workspaces__workspace_id__work_pools__work_pool_name__queues_filter_post or https://github.com/PrefectHQ/prefect/blob/6dbf284f9c99b3dfbbecf22638c5d71d621f6b93/src/prefect/server/api/workers.py#L472-L473)

At first, I just aliased mapWorkPoolQueuesFilter = mapWorkQueuesFilter and similarly for the types, and then realized i could just cut the old one out and only use 1.